### ISSUE TITLE
Issue with "<progress>: The Progress Indicator element": Image of progress bar lacks alt text

### ISSUE THREAD

**Issue by harangn** [mdn/content#14357]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress

#### What information was incorrect, unhelpful, or incomplete?
The screenshot of a Windows 7 progress bar has poor alt text which is just the image's file name: "progress-firefox.JPG"

#### Specific section or headline?
[Result heading.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress#result)

#### What did you expect to see?
A description of the image, such as: "Windows 7 progress bar screenshot. A rectangular gray bar with progress shown in green."

#### Did you test this? If so, how?
Yes, by inspecting in browser.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/html/element/progress`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/progress/index.md
* Last commit: https://github.com/mdn/content/commit/11f1e24a562dd20e7e50b98e3e66f4927dcc92a2
* Document last modified: 2022-03-15T15:33:00.000Z

</details>

**Comment by sideshowbarker** [comment]
Thanks for raising this — I’ve opened #14360 with a fix.

After that fix is merged, this issue will automatically be marked as resolved, and closed.
